README.md

```diff
@@ -211,6 +213,25 @@ $records = $porter->import(
         ->setCacheAdvice(CacheAdvice:SHOULD_CACHE())
 );
 ```
+### Cache key
+
+The cache key can optionally be generated by an implementation of `CacheKeyGeneratorInterface` passed into `CachingConnector::fetch`. This implementation should provide one method `generateCacheKey` which returns a [PSR-6][PSR-6]-compatible cache key.
```

Comment:
As mentioned, this doesn't seem to be correct any more. `CachingConnector::fetch` does not receive `CacheKeyGeneratorInterface`.